VideoLocker/src/main/java/org/edx/mobile/task/GetCommentsListTask.java

```diff
@@ -25,8 +25,15 @@ public GetCommentsListTask(@NonNull Context context, @NonNull String responseId,
     }
 
     public Page<DiscussionComment> call() throws Exception {
-        List<String> requestedFields = Collections.singletonList(
-                DiscussionRequestFields.PROFILE_IMAGE.getQueryParamValue());
+        final List<String> requestedFields;
+        {
```

Comment:
If you just want to organize it, then you could do that by grouping, comments, etc.; you don't really need to use actual code blocks.